src/replication/ng/policy/manager.go

```diff
@@ -32,3 +125,86 @@ type Manager interface {
 	// Remove the specified policy
 	Remove(int64) error
 }
+
+// DefaultManager provides replication policy CURD capabilities.
+type DefaultManager struct{}
+
+var _ Manager = &DefaultManager{}
+
+// NewDefaultManager is the constructor of DefaultManager.
+func NewDefaultManager() *DefaultManager {
+	return &DefaultManager{}
+}
+
+// Create creates a new policy with the provided data;
+// If creating failed, error will be returned;
+// If creating succeed, ID of the new created policy will be returned.
+func (m *DefaultManager) Create(policy *model.Policy) (int64, error) {
+	ply, err := convertToPersistModel(policy)
+	if err != nil {
+		return 0, err
+	}
+	return dao.AddRepPolicy(*ply)
+}
+
+// List returns all the policies
+func (m *DefaultManager) List(queries ...*model.PolicyQuery) (total int64, polices []*model.Policy, err error) {
+	// default query parameters
+	var name = ""
+	var namespace = ""
+	var page int64 = 1
+	var pageSize int64 = 15
+
```

Comment:
The default values for page and size are given on the API level, so the default values for them aren't needed here.